_posts/2020-02-23-kfserving-0.5.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Operationalize, Scale and Infuse Trust in AI Models using KFServing"
+description: "KFServing 0.5 Blog Post"
+layout: post
+toc: false
+comments: true
+image: images/logo.png
+hide: false
+categories: [release, official]
+---
+
+#### By **Animesh Singh** and **Dan Sun**
+
+With inputs from : Yuzhui Liu, Tommy Li, Paul Vaneck, Andrew Butler, Srinivasan Parthasarathy
+
+[<u>According to Gartner</u>](https://www.gartner.com/document/3994810), "organizations are now interested not just in a “pathway” to production but in a “speedway” to production. However, most enterprises become dismayed when they find out how exceptionally challenging it is to operationalize AI, as opposed to how (deceptively) easy it is to perform proof of concepts (POCs) or small-scale projects."
```

Comment:
Do we have clearance from Gartner to use this quote?

Reply:
while i have sent a note to Gartner and IDC to seek approval, for now I have removed that and rewrote the intro.